components/webdriver_server/Cargo.toml

```diff
@@ -31,4 +32,4 @@ servo_config = { path = "../config" }
 servo_url = { path = "../url" }
 style_traits = { path = "../style_traits" }
 uuid = { version = "0.8", features = ["v4"] }
-webdriver = "0.40"
+webdriver = "0.44"
```

Comment:
necessary to ensure we don't use 3 separate versions of hyper. This cuts it down to 2.
```
cargo tree -i hyper
error: There are multiple `hyper` packages in your project, and the specification `hyper` is ambiguous.
Please re-run this command with `-p <spec>` where `<spec>` is one of the following:
  hyper:0.13.10
  hyper:0.14.16
```